vllm/entrypoints/openai/protocol.py

```diff
@@ -172,20 +172,20 @@ class ChatCompletionRequest(OpenAIBaseModel):
     echo: Optional[bool] = Field(
         default=False,
         description=(
-            "If true, the new message will be prepended with the last message "
+            "If True, the new message will be prepended with the last message "
             "if they belong to the same role."),
     )
-    add_generation_prompt: Optional[bool] = Field(
+    add_generation_prompt: bool = Field(
```

Comment:
Why remove the `Optional` here but not for all the other `bool` fields?

Reply:
Let me update the rest as well.